### PR TITLE
Feature/editor history

### PR DESCRIPTION
**Pull request by TatianaLopaeva**
- Added history to the editors. 
- Added storing and highlighting changes in history.
- Added possibility to restore history.

### CODE REVIEW

File: packages/asc-web-common/api/files/index.js [hunk 1]
```diff
@@ -809,3 +809,26 @@ export function fileCopyAs(fileId, destTitle, destFolderId, enableExternalExt) {
...
+  return request({
+    method: "get",
+    url: `files/file/${fileId}/edit/history?doc=${doc}`,
+  });
+}
+
+export function getEditDiff(fileId, version, doc) {
+  return request({
+    method: "get",
+    url: `files/file/${fileId}/edit/diff?version=${version}&&doc=${doc}`,
```
Comment (AlexeySafronov) [inline packages/asc-web-common/api/files/index.js:823]: Remove double &

File: packages/asc-web-common/api/files/index.js [hunk 2]
```diff
@@ -809,3 +809,26 @@ export function fileCopyAs(fileId, destTitle, destFolderId, enableExternalExt) {
...
+  });
+}
+
+export function restoreDocumentsVersion(fileId, version, doc) {
+  const options = {
+    method: "get",
+    url: `files/file/${fileId}/restoreversion?version=${version}&&doc=${doc}`,
```
Comment (AlexeySafronov) [inline packages/asc-web-common/api/files/index.js:830]: Remove double &

File: packages/asc-web-common/api/files/index.js (hunk 1 above)
Comment (TatianaLopaeva) [reply packages/asc-web-common/api/files/index.js]: Removed

File: packages/asc-web-common/api/files/index.js (hunk 2 above)
Comment (TatianaLopaeva) [reply packages/asc-web-common/api/files/index.js]: Removed

**Review by AlexeySafronov — APPROVED** [review]

**Pull request merged**